src/Components/NavigationBar/NavigationBar.tsx

```diff
@@ -0,0 +1,31 @@
+import "./NavigationBar.css";
+import {Link} from "react-router-dom";
+
+export const NavigationBar=()=>{
+    return (
+        <nav className="component-navigation flex-end-space-betw">
+
+        <div className="navigation-pri-part">
+            <div className="logo-container"><img src={require("./../../Assets/Images/50px-50px.png")} alt="Beautiful piston"/>
+            </div>
+            <a >
```

Comment:
Try to use "Link" from react router instead of anchor tags for navigation.

Reply:
This is just for onClick not for navigation 
Thanks for reviewing though